source/agora/test/NetworkManager.d

```diff
@@ -178,6 +179,12 @@ unittest
     node_bad.clearFilter();
     node_validators.each!(node => node.clearFilter());
 
+    // The testing time decreases compared to the last code because setting
+    // seeds to the `PreImageCycle`s make it unnecessary to generate kind of
+    // large number seeds. So we have to add some preparation time to this
+    // network test like this.
+    Thread.sleep(8.seconds);
+
```

Comment:
We probably should find out why. If the setup is quicker it should not mean that we need this extra time for this test.

Reply:
Right. I will look into it and leave the `TODO` comment about that. Ty.

Reply:
Updated